modules/Microsoft.Compute/virtualMachines/readme.md

```diff
@@ -496,11 +496,11 @@ configurationProfileAssignments: [
   "value": {
     "enabled": true,
     "settings": {
-      "domainName": "contoso.com",
-      "domainJoinUser": "test.user@testcompany.com",
```

Comment:
The property on the vm module still references the old parameters names. Check "extensionDomainJoinPassword". The module and the readme need to be updated to retro fit the new changes.

Reply:
@ahmadabdalla Good catch. Updated.